app/models/user.rb

```diff
@@ -1,9 +1,8 @@
 class User < ApplicationRecord
-  has_many :transactions
+  has_many :transactions, dependent: :destroy
 
   mount_uploader :avatar, AvatarUploader
 
-  devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable, :omniauthable, :confirmable
-  devise :omniauthable, omniauth_providers: %i[google_oauth2 github]
+  devise :database_authenticatable, :registerable, :recoverable, :rememberable, :validatable
+  devise :confirmable, :omniauthable, omniauth_providers: %i[google_oauth2 github]
```

Comment:
@marderer why did you split `devise` options to two lines and moved `:confirmable` to the second line
As for we - `devise :omniauthable, omniauth_providers: %i[google_oauth2 github]` looks great